index.css

```diff
@@ -0,0 +1,247 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  background-color: #eceef1;
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 0.9em;
+}
+
+/* == header styles == */
+header {
+  padding: 0.4em;
+}
+
+nav {
+  display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Avoid using the same rule many times. Try this

```
.flex{
  display:flex;
}
```

In the HMTL file in every element that uses white color
`<div class="flex"></div>`

- Please once you create the class, delete these rules in the entire CSS file


###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_